apps/assistant-service/Dockerfile

```
@@ -8,17 +8,17 @@ COPY gradle ./gradle
 # common-libs 복사
 COPY common ./common
 
-# ai-service 소스 복사
-COPY apps/ai-service ./apps/ai-service
+# assistant-service 소스 복사
+COPY apps/assistant-service ./apps/assistant-service
 
 RUN chmod +x gradlew
-RUN ./gradlew :apps:ai-service:clean :apps:ai-service:build -x test --no-daemon
+RUN ./gradlew :apps:assistant-service:clean :apps:assistant-service:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)